deployment/deploy-presidio.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-REGISTRY=${1:-presidio.azurecr.io}
+REGISTRY=${1:-mcr.microsoft.com/presidio/}
```

Comment:
all pushes will be done to mcr? will this be controllable?
what permission is required to commit to mcr?

Reply:
should be:
mcr.microsoft.com